--- a/schema.rs
+++ b/schema.rs
@@ -1,82 +1,82 @@
 // @generated automatically by Diesel CLI.
 
 diesel::table! {
     author (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         name -> Text,
     }
 }
 
 diesel::table! {
     committer (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         name -> Text,
     }
 }
 
 diesel::table! {
     dependency (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         dependency_type_id -> BigInt,
         name -> Text,
     }
 }
 
 diesel::table! {
     dependency_module (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         dependency_id -> BigInt,
         module_id -> BigInt,
     }
 }
 
 diesel::table! {
     dependency_osv (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         dependency_module_id -> BigInt,
         osv_id -> Text,
         details -> Text,
         fixed_in -> Text,
     }
 }
 
 diesel::table! {
     dependency_type (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         name -> Text,
     }
 }
 
 diesel::table! {
     gh_organization (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         name -> Text,
     }
 }
 
 diesel::table! {
     gh_repository (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         name -> Text,
         gh_organization_id -> BigInt,
         create_date -> Text,
         update_date -> Text,
     }
 }
 
 diesel::table! {
     maintainer (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         name -> Text,
     }
 }
 
 diesel::table! {
     module (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         technical_name -> Text,
         version_odoo -> Integer,
         name -> Text,
         version_module -> Text,
         description -> Nullable<Text>,
         website -> Nullable<Text>,
@@ -85,56 +85,56 @@
         auto_install -> Bool,
         application -> Bool,
         installable -> Bool,
         gh_repository_id -> BigInt,
         create_date -> Text,
         update_date -> Text,
-        folder_size -> Integer,
+        folder_size -> BigInt,
         last_commit_hash -> Text,
         last_commit_author -> Text,
         last_commit_name -> Text,
         last_commit_date -> Text,
         last_commit_partof -> Nullable<Text>,
         installation -> Nullable<Text>,
         usage -> Nullable<Text>,
     }
 }
 
 diesel::table! {
     module_author (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         module_id -> BigInt,
         author_id -> BigInt,
     }
 }
 
 diesel::table! {
     module_committer (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         module_id -> BigInt,
         committer_id -> BigInt,
         commits -> Integer,
         insertions -> Integer,
         deletions -> Integer,
     }
 }
 
 diesel::table! {
     module_committer_period (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         module_id -> BigInt,
         committer_id -> BigInt,
         year -> Integer,
         month -> Integer,
         commits -> Integer,
     }
 }
 
 diesel::table! {
     module_controller (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         module_id -> BigInt,
         class_name -> Text,
         name -> Text,
         routes -> Text,
         auth -> Nullable<Text>,
         http_type -> Text,
@@ -147,21 +147,21 @@
         module_version_id -> BigInt,
     }
 }
 
 diesel::table! {
     module_maintainer (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         module_id -> BigInt,
         maintainer_id -> BigInt,
     }
 }
 
 diesel::table! {
     module_model (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         module_id -> BigInt,
         model_name -> Text,
         class_name -> Text,
         inherit_from -> Nullable<Text>,
         is_new_model -> Bool,
         docstring -> Nullable<Text>,
@@ -169,119 +169,119 @@
         module_version_id -> BigInt,
     }
 }
 
 diesel::table! {
     module_model_field (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         module_model_id -> BigInt,
         name -> Text,
         field_type -> Text,
         relation -> Nullable<Text>,
         attrs -> Nullable<Text>,
     }
 }
 
 diesel::table! {
     module_model_method (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         module_model_id -> BigInt,
         name -> Text,
         decorators -> Nullable<Text>,
         signature -> Text,
         docstring -> Nullable<Text>,
     }
 }
 
 diesel::table! {
     module_record (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         module_id -> BigInt,
         xml_id -> Text,
         model -> Text,
         noupdate -> Bool,
         fields -> Nullable<Text>,
         module_version_id -> BigInt,
     }
 }
 
 diesel::table! {
     module_security_warning (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         module_id -> BigInt,
         severity -> Text,
         code -> Text,
         message -> Text,
         xml_id -> Nullable<Text>,
         module_version_id -> BigInt,
     }
 }
 
 diesel::table! {
     module_version (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         module_id -> BigInt,
         version_module -> Text,
         create_date -> Text,
         update_date -> Text,
     }
 }
 
 diesel::table! {
     module_view (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         module_id -> BigInt,
         xml_id -> Text,
         name -> Nullable<Text>,
         model -> Nullable<Text>,
         inherit_xml_id -> Nullable<Text>,
         view_type -> Nullable<Text>,
         module_version_id -> BigInt,
     }
 }
 
 diesel::table! {
     pull_request (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         name -> Text,
         version_odoo -> Integer,
         module_technical_name -> Text,
-        prid -> Integer,
+        prid -> BigInt,
         gh_repository_id -> BigInt,
         created_at -> Nullable<Text>,
         ci_status -> Nullable<Text>,
         last_message_at -> Nullable<Text>,
     }
 }
 
 diesel::table! {
     pull_request_history (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         module_technical_name -> Text,
         version_odoo -> Integer,
         gh_repository_id -> BigInt,
-        prid -> Integer,
+        prid -> BigInt,
         created_at -> Text,
         closed_at -> Text,
     }
 }
 
 diesel::table! {
     system_event (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         message -> Text,
         date -> Text,
         event_type_id -> BigInt,
         severity -> Text,
         is_html -> Bool,
     }
 }
 
 diesel::table! {
     system_event_type (id) {
-        id -> Nullable<BigInt>,
+        id -> BigInt,
         name -> Text,
     }
 }
 
 diesel::joinable!(module_controller -> module_version (module_version_id));
 diesel::joinable!(module_model -> module_version (module_version_id));
